<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>278</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>96</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="selectBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>IP:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ipText"/>
        </item>
        <item>
         <widget class="QLabel" name="portLabel">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="portBox"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Minimum Value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="minTxt"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Maximum Value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maxTxt"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Threshold:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="threshold"/>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="datapath"/>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="heightLabel">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="widthBox"/>
        </item>
        <item>
         <widget class="QLabel" name="widthLabel">
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="heightBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Format:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="formatBox"/>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>History Data Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnDataLoad">
          <property name="text">
           <string>load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Configure Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnCfgLoad">
          <property name="text">
           <string>load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="0">
       <widget class="QLineEdit" name="cfgpath"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="parametersBox">
     <property name="title">
      <string>Description</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="7" column="0">
       <widget class="QLabel" name="manufacturerLabel">
        <property name="text">
         <string>Manufacturer:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="vidLabel">
        <property name="text">
         <string>Vendor ID:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="serialNumberLabel">
        <property name="text">
         <string>Serial number:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="pidLabel">
        <property name="text">
         <string>Product ID:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
